mysql-test/suite/innodb/t/MDEV-28315.test

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE t1 (a VARBINARY(128)) CHARACTER SET utf32;
+INSERT INTO t1 VALUES ('South Carolina, Vermont, New Jersey, New Mexico, Wisconsin, Missouri, Delaware');
+CREATE TABLE t2 (b SET('South Carolina', 'Vermont', 'Texas', 'New Mexico', 'Wisconsin', 'Missouri', 'Delaware', 'Wyoming', 'New Jersey', 'Maryland', 'Illinois', 'New York')) CHARACTER SET utf32;
+--error WARN_DATA_TRUNCATED
+INSERT INTO t2 SELECT * FROM t1;
```

Comment:
No newline at end of the line

Reply:
There should be a `SELECT * FROM t2` at the end here, to explain and verify the expected contents of `t2` if the memory bug is not present.

I expected the corresponding result would just be:
```
b
South Carolina
```

Reply:
Thanks. I added the statement. The output was just `b` so looks like that `b` is an empty table. I talked with @nayuta-yanagisawa and concluded it is expected as a result of truncation.